src/shared/lib/CivicUtils.ts

```diff
@@ -134,3 +135,21 @@ export function buildCivicEntry(geneEntry: ICivicGeneData, geneVariants: {[name:
         variants: geneVariants
     };
 }
+
+export function getCivicCNAVariants(copyNumberData:DiscreteCopyNumberData[], geneSymbol: string, civicVariants:ICivicVariant): {[name: string]: ICivicVariantData} {
+    let geneVariants: {[name: string]: ICivicVariantData} = {};
+    if (copyNumberData[0].alteration == 2) {
+        for (let alteration in civicVariants[geneSymbol]) {
+            if (alteration == "AMPLIFICATION") {
```

Comment:
In general, `===` is safer than `==`

Reply:
@oplantalech Yes I mentioned this on a previous PR as well I believe, I don't think there's really any reason to use `==` in JS, always use `===`